### ISSUE TITLE
fuse server daemon not getting parallel requests even though multithreaded option selected

### ISSUE THREAD

**Issue by asunderramesh** [osxfuse/osxfuse#161]
The fuse user daemon is not getting parallel requests from fuse library, even though the fuse is configured as multithreaded. From the logs, I could observe that requests(open, read, write, attributes etc..) are coming from multiple threads but when one thread/request is taking more time in the fuse server user mode daemon implementation (making a http call or some other operation), the user mode daemon is not receiving any other requests till the time it completes the previous request. 
In short, though requests are coming from multiple threads, at a time only one request is coming to the fuse user mode program. I am using osx 10.7 and osxfuse 2.7.1.

**Comment by geofflu** [comment]
Have you found a positive response to your issue ? I actually have the same problem and i have no idea how to deal with it.

Thanks in advance for your reply
